common/static/common/js/components/views/feedback_notification.js

```diff
@@ -1,34 +1,34 @@
-;(function (define) {
+;(function(define) {
     'use strict';
     define(["jquery", "underscore", "underscore.string", "common/js/components/views/feedback"],
         function($, _, str, SystemFeedbackView) {
 
-        var Notification = SystemFeedbackView.extend({
-            options: $.extend({}, SystemFeedbackView.prototype.options, {
+            var Notification = SystemFeedbackView.extend({
+                options: $.extend({}, SystemFeedbackView.prototype.options, {
                 type: "notification",
-                closeIcon: false
-            })
-        });
+                    closeIcon: false
+                })
+            });
 
-        // create Notification.Warning, Notification.Confirmation, etc
-        var capitalCamel, intents;
-        capitalCamel = _.compose(str.capitalize, str.camelize);
+            // create Notification.Warning, Notification.Confirmation, etc
+            var capitalCamel, intents;
+            capitalCamel = _.compose(str.capitalize, str.camelize);
         intents = ["warning", "error", "confirmation", "announcement", "step-required", "help", "mini"];
```

Comment:
Thanks. I'd tried to revert my changes in this file, but PyCharm fooled me as I had it in 'ignore whitespace' mode. I've fully reverted them now.